src/pages/tutorial/angular/step-4.mdx

```diff
@@ -0,0 +1,620 @@
+---
+title: 4. Creating components
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### With two pages comprised entirely of Carbon components, let's revisit the landing page and build a couple components of our own by using Carbon icons and tokens.
+
+<AnchorLinks>
+
+<AnchorLink>Fork, clone and branch</AnchorLink>
+<AnchorLink>Review design</AnchorLink>
+<AnchorLink>Create components</AnchorLink>
+<AnchorLink>Use components</AnchorLink>
+<AnchorLink>Add styling</AnchorLink>
+<AnchorLink>Check accessibility</AnchorLink>
+<AnchorLink>Submit pull request</AnchorLink>
+
+</AnchorLinks>
+
+## Preview
+
+Carbon provides a solid foundation for building web applications through its color palette, layout, spacing, type, as well as common building blocks in the form of components. So far, we've only used Carbon components to build out two pages.
+
+Next, we're going to use Carbon assets to build application-specific components. We'll create our components with considerations for accessibility and responsiveness.
+
+A [preview](https://angular-step-5--carbon-tutorial.netlify.com) of what you'll build (see bottom of page):
```

Comment:
![image](https://user-images.githubusercontent.com/1691245/67427030-e78f2000-f5a0-11e9-958e-c6c7feb687bf.png)

Reply:
(Have extra `-` in URL.)

Reply:
Fixed